Apps/Src/ArrayConnection.c

```diff
@@ -62,16 +62,10 @@ void Task_ArrayConnection(void *p_arg) {
 
         if (desiredState == ON && currentState != ON) {
```

Comment:
The only task I can think of that activates the array is read-switches and that hasn't been merged in yet. Will this task work without it? I see that we're accessing car_state->ShouldArrayBeActivated, and that's not initialized anywhere.